decompiler/config/jak2/jak2_config.jsonc

```diff
@@ -139,6 +139,15 @@
       "hacks_merge_file": "decompiler/config/jak2/pal/hacks.jsonc",
       "obj_file_name_map_file": "goal_src/jak2/build/all_objs.json",
       "object_patches": {}
+    },
+    "ntsc_v2": {
+      "game_name": "jak2_us2",
+      "type_casts_merge_file": "decompiler/config/jak2/pal/type_casts.jsonc",
+      "label_types_merge_file": "decompiler/config/jak2/pal/label_types.jsonc",
+      "hacks_merge_file": "decompiler/config/jak2/pal/hacks.jsonc",
+      "anonymous_function_types_merge_file": "decompiler/config/jak2/pal/anonymous_function_types.jsonc",
+      "obj_file_name_map_file": "goal_src/jak2/build/all_objs.json",
+      "object_patches": {}
     }
```

Comment:
so i was mistaken, and original PAL doesn't have the fix.  Apparently someone reproduced the problem on original PAL (which makes sense, unlike Jak 1, Jak 2 released in EU mere days later).

You mentioned this is the only difference across all versions, and it isn't PAL specific.  If it's shared for both ntsc_v2 and PAL_v2, perhaps it should go in a better named folder like `ntsc_pal_v2`.

Reply:
Ah ok. I read at first that PAL had the fix so I just assumed.

Either way I'll be comparing all 5 version now.